app/helpers/fileUpload/FileUploadGenericErrorReporter.scala

```diff
@@ -101,6 +115,7 @@ object FileUploadGenericErrorReporter {
     case EventType.Event6 => eventHeader(eventType)
     case EventType.Event22 => eventHeader(eventType)
     case EventType.Event23 => eventHeader(eventType)
+    case EventType.Event24 => eventHeader(eventType)
```

Comment:
Can we aggregate the cases here? `case Event6 | Event22 | ... => eventHeader(eventType)`